src/DssSpell.sol

```diff
@@ -54,32 +61,37 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     //
 
     function officeHours() public override returns (bool) {
-        return true;
+        return false;
     }
 
     function actions() public override {
 
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        onboardNewCollaterals();
+        // onboardNewCollaterals();
         // offboardCollaterals();
 
-        drawFromRWA009Urn();
+        address RWA009_A_URN = DssExecLib.getChangelogAddress("RWA009_A_URN");
+        address MCD_VEST_DAI = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
 
-        // Add RWA_TOKEN_FAB to changelog
-        DssExecLib.setChangelogAddress("RWA_TOKEN_FAB", RWA_TOKEN_FAB);
-
-        DssExecLib.setChangelogVersion("1.13.3");
-    }
+        // Huntingdon Valley (HVBank) Vault Drawdown
+        RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
 
-    function drawFromRWA009Urn() internal {
-        // lock RWA009 Token in the URN
-        ERC20Like(RWA009).approve(RWA009_A_URN, 1 * WAD);
-        RwaUrnLike(RWA009_A_URN).lock(1 * WAD);
 
-        // draw DAI to genesis address
-        RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
+        // Keep3r Network Stream Re-Deployment
+        VestLike(MCD_VEST_DAI).yank(8);
+        VestLike(MCD_VEST_DAI).restrict(
+            VestLike(MCD_VEST_DAI).create({
+                usr: 0x37b375e3D418fbECba6b283e704F840AB32f3b3C,
+                tot: 215_000 * WAD,
+                bgn: 1656633600,
+                tau: 18489600,
```

Comment:
NIT: Could we replace this for:
```suggestion
                bgn: JUL_01_2022,
                tau: FEB_01_2023 - JUL_01_2022,
```
?
(ofc declaring both dates on top). This is not a hard requirement and at the end up to the spell writer but it is so far how we've been doing it and seems a bit easier to check.